#!/usr/bin/env python
"""
    AIS CONFIDENTIAL
    Author: Rakesh Shrestha (r.shrestha@ai-systems.ca)
"""

PACKAGE = "ais_state_machine"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("MANUAL_SAFETY",   bool_t,   0, "Whether safety enabled in manual mode",  False)
exit(gen.generate(PACKAGE, "ais_state_machine", "DynamicParams"))
